aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/APIGatewayV2ProxyRequestEvent.java

```diff
@@ -1,726 +1,4 @@
-  }
-
-  public void setMultiValueHeaders(Map<String, List<String>> multiValueHeaders) {
-    this.multiValueHeaders = multiValueHeaders;
-  }
-
-  public Map<String, String> getQueryStringParameters() {
-    return queryStringParameters;
-  }
-
-  public void setQueryStringParameters(Map<String, String> queryStringParameters) {
-    this.queryStringParameters = queryStringParameters;
-  }
-
-  public Map<String, List<String>> getMultiValueQueryStringParameters() {
-    return multiValueQueryStringParameters;
-  }
-
-  public void setMultiValueQueryStringParameters(Map<String, List<String>> multiValueQueryStringParameters) {
-    this.multiValueQueryStringParameters = multiValueQueryStringParameters;
-  }
-
-  public Map<String, String> getPathParameters() {
-    return pathParameters;
-  }
-
-  public void setPathParameters(Map<String, String> pathParameters) {
-    this.pathParameters = pathParameters;
-  }
-
-  public Map<String, String> getStageVariables() {
-    return stageVariables;
-  }
-
-  public void setStageVariables(Map<String, String> stageVariables) {
-    this.stageVariables = stageVariables;
-  }
-
-  public RequestContext getRequestContext() {
-    return requestContext;
-  }
-
-  public void setRequestContext(RequestContext requestContext) {
-    this.requestContext = requestContext;
-  }
-
-  public String getBody() {
-    return body;
-  }
-
-  public void setBody(String body) {
-    this.body = body;
-  }
-
-  public boolean isIsBase64Encoded() {
-    return isBase64Encoded;
-  }
-
-  public void setIsBase64Encoded(boolean isBase64Encoded) {
-    this.isBase64Encoded = isBase64Encoded;
-  }
-
-  @Override
-  public int hashCode() {
-    int hash = 7;
-
-    hash = 43 * hash + Objects.hashCode(this.requestContext);
-    hash = 43 * hash + Objects.hashCode(this.body);
-    hash = 43 * hash + (this.isBase64Encoded ? 1 : 0);
-
-    return hash;
-  }
-
-  @Override
-  public boolean equals(Object obj) {
-    if (this == obj) {
-      return true;
-    }
-
-    if (obj == null) {
-      return false;
-    }
-
-    if (getClass() != obj.getClass()) {
-      return false;
-    }
-
-    final APIGatewayV2ProxyRequestEvent other = (APIGatewayV2ProxyRequestEvent) obj;
-
-    if (this.isBase64Encoded != other.isBase64Encoded) {
-      return false;
-    }
-
-    if (!Objects.equals(this.body, other.body)) {
-      return false;
-    }
-
-    if (!Objects.equals(this.requestContext, other.requestContext)) {
-      return false;
-    }
-    return true;
-  }
-
-  @Override
-  public String toString() {
-    StringBuilder sb = new StringBuilder();
-    sb.append("{");
-
-    if (requestContext != null) {
-      sb.append("requestContext: ").append(requestContext).append(",");
-    }
-
-    if (body != null) {
-      sb.append("body: ").append(body).append(",");
-    }
-
-    sb.append("isBase64Encoded: ").append(isBase64Encoded).append(",");
-
-    sb.append("}");
-
-    return sb.toString();
-  }
-
-}
+@Deprecated()
```

Comment:
Maybe a message in the `@Deprecated` tag could be helpful in letting users know why the class is deprecated and what they use instead?